src/app/service/sing-in/sign-in.service.ts

```diff
@@ -5,19 +5,25 @@ import { User } from '../../model';
 @Injectable({
   providedIn: 'root'
 })
-export class SingInService {
+export class SignInService {
 
   private currentUserSbj = new BehaviorSubject<User | null>(null);
 
   constructor() { }
 
-  login(data?: User) {
-
+  login(data: User) {
     return this.currentUserSbj.next(data as User);
   }
 
+  logout() {
+    return this.currentUserSbj.next(null);
+  }
+
   public get isUserLoggedIn(): boolean {
     return this.currentUserSbj.getValue() ? true : false;
   }
-}
 
+  public get userLoggedInData() {
```

Comment:
But it's not a data, it's the subject as observable

Reply:
Fixed